packages/cli-lib/lang/en.lyaml

```diff
@@ -907,8 +898,33 @@ en:
         options:
           options:
             describe: "Options to pass to javascript fields files"
-
     lib:
+      projects:
```

Comment:
I cleaned up the lang file a little bit. There were a handful of keys in here that weren't in the right section. We have a dedicated `lib` category for every util in the lib folder.